apps/extension/src/routes/page/router.tsx

```diff
@@ -23,8 +21,19 @@ export const pageRoutes: RouteObject[] = [
         element: <Onboarding />,
         children: onboardingRoutes,
       },
+      {
+        path: PagePath.RESTORE_PASSWORD,
+        element: <RestorePasswordIndex />,
+        children: restorePasswordRoutes,
+        loader: noPasswordIndexLoader,
+      },
+      {
+        path: PagePath.LOGIN,
+        element: <Login />,
+        loader: noPasswordIndexLoader,
+      },
```

Comment:
These loaders are another case we don't anticipate and I don't think we need to guard for.

Reply:
I don't understand why we have to remove noPasswordIndexLoader and onboardingIndexLoader! a user without them can enter links and be thrown to any link he shouldn't see! for example, a logged in user can go to the login page

Reply:
I understood that you used keplr as a basis and you can add a few seed phrases to them